portal-bridge/src/bridge.rs

```diff
@@ -79,9 +79,19 @@ impl Bridge {
     }
 
     async fn launch_test(&self, test_path: PathBuf) {
+        let extension = test_path
+            .extension()
+            .and_then(|x| x.to_str())
```

Comment:
Let's avoid `x` as a variable name, something like `path` would be fine and short

Reply:
resolved